koku/masu/database/trino_sql/reporting_awscostentrylineitem_summary_by_ec2_compute.sql

```diff
@@ -0,0 +1,133 @@
+INSERT INTO postgres.{{schema | sqlsafe}}.reporting_awscostentrylineitem_summary_by_ec2_compute (
+    uuid,
+    usage_start,
+    usage_end,
+    usage_account_id,
+    resource_id,
+    instance_name,
+    instance_type,
+    operating_system,
+    region,
+    vcpu,
+    memory,
+    tags,
+    cost_category,
+    unit,
+    usage_amount,
+    normalization_factor,
+    normalized_usage_amount,
+    currency_code,
+    unblended_rate,
+    unblended_cost,
+    markup_cost,
+    blended_rate,
+    blended_cost,
+    markup_cost_blended,
+    savingsplan_effective_cost,
+    markup_cost_savingsplan,
+    calculated_amortized_cost,
+    markup_cost_amortized,
+    public_on_demand_cost,
+    public_on_demand_rate,
+    source_uuid,
+    cost_entry_bill_id,
+    account_alias_id
+)
+with cte_pg_enabled_keys as (
+    select array_agg(key order by key) as keys
+      from postgres.{{schema | sqlsafe}}.reporting_enabledtagkeys
+     where enabled = true
+     and provider_type = 'AWS'
+)
+SELECT uuid() as uuid,
+    usage_start,
+    usage_end,
+    cast(usage_account_id AS varchar(50)),
+    resource_id,
+    instance_name,
+    instance_type,
+    operating_system,
+    region,
+    vcpu,
+    memory,
+    cast(
+        map_filter(
+            cast(json_parse(tags) as map(varchar, varchar)),
+            (k,v) -> contains(pek.keys, k)
+        ) as json
+     ) as tags,
+    json_parse(costcategory) as cost_category,
+    unit,
+    cast(usage_amount AS decimal(24,9)),
+    normalization_factor,
+    normalized_usage_amount,
+    cast(currency_code AS varchar(10)),
+    cast(unblended_rate AS decimal(24,9)),
+    cast(unblended_cost AS decimal(24,9)),
+    cast(unblended_cost * {{markup | sqlsafe}} AS decimal(24,9)) as markup_cost,
+    cast(blended_rate AS decimal(24,9)),
+    cast(blended_cost AS decimal(24,9)),
+    cast(blended_cost * {{markup | sqlsafe}} AS decimal(33,15)) as markup_cost_blended,
```

Comment:
Is there any reason these decimals aren't consistent? we have `(24,9)`,  `(33,15)` and `(33,9)` present even though these fields are pretty similar.

Reply:
These are consistent across the different aws tables where we have these columns; https://github.com/project-koku/koku/blob/main/koku/reporting/provider/aws/models.py. 
I assumed we have a good reason for the variation.